cmd/wazero/wazero.go

```diff
@@ -231,11 +243,17 @@ func doRun(args []string, stdOut io.Writer, stdErr logging.Writer, exit func(cod
 	} else if needsGo {
 		gojs.MustInstantiate(ctx, rt)
 		err = gojs.Run(ctx, rt, code, conf)
+	} else {
+		_, err = rt.InstantiateModule(ctx, code, conf)
 	}
 
 	if err != nil {
 		if exitErr, ok := err.(*sys.ExitError); ok {
-			exit(int(exitErr.ExitCode()))
+			exitCode := exitErr.ExitCode()
+			if timeout > 0 && exitCode == sys.ExitCodeDeadlineExceeded {
```

Comment:
do we need `timeout >0`? I guess `sys.ExitCodeDeadlineExceeded` won't be set if `timeout == 0`